extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -1054,7 +1059,9 @@ class CornerstoneViewportService extends PubSubService implements IViewportServi
       // Reset the camera for viewports that should reset their camera on resize,
       // which means only those viewports that have a zoom level of 1.
```

Comment:
Can you fix this comment - we reset the camera on resize for all viewports now, using the position presentation to maintain the same relative size/position.